### PR TITLE
Update cypress test. Create project fail. Update project fail. Delete project fail.

### PR DESCRIPTION
**Pull request by DmitriyOparin**
Update cypress test base_actions_project_task_user.js new steps.
Add new className for elements.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js [hunk 1]
```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
...
             cy.get('.cvat-constructor-viewer').should('not.exist');
         });
-        it('Logout first user, register second user, logout.', () => {
+        it('Logout first user, register second user, tries to create project and logout.', () => {
             cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
```
Comment (bsekachev) [inline tests/cypress/integration/actions_projects/base_actions_project_task_user.js:120]: Is this window expected to be visible after each logout? Maybe put it inside logout?

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js [hunk 2]
```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
...
+            cy.get('.cvat-attribute-type-input').click();
+            cy.get('.cvat-attribute-type-input-text').click();
+            cy.get('[placeholder="Default value"]').type(textDefaultValue);
+            if (multiAttrParams) {
+                cy.updateAttributes(multiAttrParams);
+            }
+            cy.contains('button', 'Done').click();
+            cy.get('.cvat-create-project-content').within(() => {
+                cy.contains('Submit').click();
+            });
```
Comment (bsekachev) [inline tests/cypress/integration/actions_projects/base_actions_project_task_user.js:141]: Looks like cypress command: ``createProject``

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js [hunk 3]
```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
...
+            cy.get('[placeholder="Label name"]').type(labelName);
+            cy.get('.cvat-new-attribute-button').click();
+            cy.get('[placeholder="Name"]').type(attrName);
+            cy.get('.cvat-attribute-type-input').click();
+            cy.get('.cvat-attribute-type-input-text').click();
+            cy.get('[placeholder="Default value"]').type(textDefaultValue);
+            if (multiAttrParams) {
+                cy.updateAttributes(multiAttrParams);
+            }
+            cy.contains('button', 'Done').click();
```
Comment (bsekachev) [inline tests/cypress/integration/actions_projects/base_actions_project_task_user.js:138]: Looks like cypress command: ``createLabel``

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js [hunk 4]
```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
...
+            cy.get('.cvat-create-project-content').within(() => {
+                cy.contains('Submit').click();
+            });
+            cy.contains('The project has been created').should('not.exist');
+            cy.get('.cvat-notification-notice-create-project-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-create-project-failed');
```
Comment (bsekachev) [inline tests/cypress/integration/actions_projects/base_actions_project_task_user.js:144]: ``closeNotification()`` also has ``get(.class-name)``. It is why we do not need get before ``closeNotification``. Right?

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js (hunk 1 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_projects/base_actions_project_task_user.js]: These actions moved in cy.logout().

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js (hunk 4 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_projects/base_actions_project_task_user.js]: Unnecessary actions removed.

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js (hunk 2 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_projects/base_actions_project_task_user.js]: Fixed.

File: tests/cypress/integration/actions_projects/base_actions_project_task_user.js (hunk 3 above)
Comment (DmitriyOparin) [reply tests/cypress/integration/actions_projects/base_actions_project_task_user.js]: Fixed.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**